docs/04-guides/get-viewer-engagement/_tabs.mdx

```diff
@@ -31,8 +31,8 @@ you.
 
 ### Caveats
 
-The viewership API is still in development. Currently, we only support start
-views, views when you open or start watching an asset. In the future, we'll
+The viewership API is still in development. Currently, we only support unique start
+views, views when a unique viewer opens or start watching an asset. In the future, we'll
```

Comment:
> views, views when a unique viewer opens or start watching an asset. In the future, we'll

small change to:

views, views when a unique viewer opens or **starts** watching an asset.